From 22432251ee04597e0fcda2e4ee6114992430e74b Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Mon, 22 Sep 2014 13:51:31 +0100
Subject: [PATCH 317/351] ASoC: arizona: Fix output sample rate enum

The output sample rate enum was referring to the wrong set of values
causing it to point to a different rate than that requested.

Change-Id: I4980a19039911810dd5c35554f57983d7c2e7001
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
---
 sound/soc/codecs/arizona.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/arizona.c b/sound/soc/codecs/arizona.c
index 0fab8a7..8541314 100644
--- a/sound/soc/codecs/arizona.c
+++ b/sound/soc/codecs/arizona.c
@@ -576,7 +576,7 @@ const struct soc_enum arizona_output_rate =
 			      0x0f,
 			      ARIZONA_OUT_RATE_ENUM_SIZE,
 			      arizona_rate_text,
-			      arizona_sample_rate_val);
+			      arizona_rate_val);
 EXPORT_SYMBOL_GPL(arizona_output_rate);
 
 const SOC_ENUM_SINGLE_DECL(arizona_spdif_rate,
-- 
1.7.9.5

